tests/baselines/reference/prototypePropertyAssignmentMergeWithInterfaceMethod.types

```diff
@@ -40,14 +40,14 @@ lf.Transaction = function() {};
  * @return {!IThenable}
  */
 lf.Transaction.prototype.begin = function(scope) {};
->lf.Transaction.prototype.begin = function(scope) {} : (scope: Array<lf.schema.Table>) => any
+>lf.Transaction.prototype.begin = function(scope) {} : (scope: Array<any>) => any

```

Comment:
Is it a regression?

Reply:
`lf.schema.Table` doesn't actually exist in the test, so we were preserving a node that failed to resolve anyway. Technically we could keep it (since it's an error anyway), but since i changed the default for jsdoc type which fail to resolve in not-jsdoc to serialize the resolved type, it now prints back as the `any` the failed resolution returns.